src/lint.rs

```diff
@@ -252,8 +334,8 @@ where
 ///
 /// - `code` is the source code in question, for example as read from a
 ///   file
-pub fn lint(code: &[u8]) -> Result<Vec<LintMatch>> {
-    lint_custom(code, builtin_lints())
+pub fn lint(code: &[u8], lint_opts: &LintOpts) -> Result<Vec<LintMatch>> {
+    lint_custom_opts(code, builtin_lints(), lint_opts)
```

Comment:
Let's please leave this function as it was. If users want to provide options they should switch to `lint_custom_opts`.